Highlight the "prompt generator" component (#39)
greenelab/manubot-gpt-manuscript: content/03.methods.md

![
**AI-based revision applied on a Manubot-based manuscript.**
**a)** A manuscript (written with Manubot) with different sections.

> **b)** The prompt generator pulls metadata from the manuscript to generate section-specific prompts for the revision of each paragraph.

Reviewer: ```suggestion
**b)** The prompt generator integrates metadata using prompt templates to generate section-specific prompts for each paragraph.
```
